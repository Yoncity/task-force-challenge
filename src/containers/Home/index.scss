@import '../../colors/index.scss';

.home_container {
  background: $primary;
  width: 100%;
}
