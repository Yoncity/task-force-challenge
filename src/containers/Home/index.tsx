import React from "react";
import "./index.scss";
import AboutMe from "../../components/AboutMe";
import Details from "../../components/Details";
import Footer from "../../components/Footer";
import ScrollableContent from "../../components/ScrollableContent";
import UserInput from "../../components/UserInput";
import Header from "../../components/Header";

const Home = () => {
  return (
    <div className="home_container">
      <Header />
      <UserInput />
      <Details />
      <ScrollableContent />
      <AboutMe />
      <Footer />
    </div>
  );
};

export default Home;
