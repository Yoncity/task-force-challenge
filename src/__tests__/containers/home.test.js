import "../../setupTests.ts";
import React from "react";
import Home from "../../containers/Home";
import configureMockStore from "redux-mock-store";
import thunk from "redux-thunk";
// import { Provider } from "react-redux";
import { shallow } from "enzyme";
import { Provider } from "react-redux";

const mockStore = configureMockStore([thunk]);
let store;

beforeEach(() => {
  store = mockStore({});
});

describe("Home Container", () => {
  it("Should Render Homepage", () => {
    const component = shallow(
      <Home />
      // <Provider store={store}>

      // </Provider>
    );
    expect(component).toMatchSnapshot();
  });
});

afterEach(() => {});
