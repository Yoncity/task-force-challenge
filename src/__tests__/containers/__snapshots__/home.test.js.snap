// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Container Should Render Homepage 1`] = `ShallowWrapper {}`;
